azure-functions/number-generator-function/package.json

```diff
@@ -7,8 +7,8 @@
     "lint": "eslint ./"
   },
   "dependencies": {
-    "axios": "^1.6.5",
```

Comment:
Small nitpick that's irrelevant for this PR but I've found it easier to separate package updates (even minor) from code changes. I've been bitten by bad package updates not being 'minor' enough in the past

Reply:
Good shout